#include "mainwindow.h"
#include "./ui_mainwindow.h"

MainWindow::MainWindow(QWidget *parent)
    : QMainWindow(parent)
    , ui(new Ui::MainWindow)
{
    ui->setupUi(this);
    ui->pushButton->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_2->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_3->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_4->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_5->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_6->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_7->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_8->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_9->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_10->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_11->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_12->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_13->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_14->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_15->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_16->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_17->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_18->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_19->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_20->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_21->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_22->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_23->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_24->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_25->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_26->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_27->setIcon(QIcon("://Images/Icon.png"));
    ui->pushButton_28->setIcon(QIcon("://Images/Icon.png"));
}

MainWindow::~MainWindow()
{
    delete ui;
}


void MainWindow::on_pushButton_clicked()
{

}


void MainWindow::on_pushButton_2_clicked()
{

}


void MainWindow::on_pushButton_3_clicked()
{

}


void MainWindow::on_pushButton_4_clicked()
{

}


void MainWindow::on_pushButton_5_clicked()
{


}


void MainWindow::on_pushButton_6_clicked()
{

}


void MainWindow::on_pushButton_7_clicked()
{

}


void MainWindow::on_pushButton_8_clicked()
{

}


void MainWindow::on_pushButton_9_clicked()
{

}


void MainWindow::on_pushButton_10_clicked()
{

}


void MainWindow::on_pushButton_11_clicked()
{

}


void MainWindow::on_pushButton_12_clicked()
{

}


void MainWindow::on_pushButton_13_clicked()
{

}


void MainWindow::on_pushButton_14_clicked()
{

}


void MainWindow::on_pushButton_15_clicked()
{

}


void MainWindow::on_pushButton_16_clicked()
{

}


void MainWindow::on_pushButton_17_clicked()
{

}


void MainWindow::on_pushButton_18_clicked()
{

}


void MainWindow::on_pushButton_19_clicked()
{

}


void MainWindow::on_pushButton_20_clicked()
{

}


void MainWindow::on_pushButton_21_clicked()
{

}


void MainWindow::on_pushButton_22_clicked()
{

}


void MainWindow::on_pushButton_23_clicked()
{

}


void MainWindow::on_pushButton_24_clicked()
{

}


void MainWindow::on_pushButton_25_clicked()
{

}


void MainWindow::on_pushButton_26_clicked()
{

}


void MainWindow::on_pushButton_27_clicked()
{

}


void MainWindow::on_pushButton_28_clicked()
{

}

